src/Views/Shared/Display/List.cshtml

```diff
@@ -42,7 +42,7 @@
         <dt class="govuk-list--description__label">Financial support</dt>
         <dd>@item.FundingOptions()</dd>
         @if (item.DistanceAddress != null || item.ProviderLocation != null) {
-          <dt class="govuk-list--description__label">Address</dt>
+          <dt class="govuk-list--description__label">Nearest Location</dt>
```

Comment:
It’s only the nearest location if there’s a distance, otherwise it’s the provider’s address

Reply:
In the latest iteration of the sql I think we still can't tell which address it was https://github.com/DFE-Digital/search-and-compare-api/pull/76/files#diff-3cf5aa843f08f6dd989cc91ef667b1d1